nixos/tests/web-apps/sharkey.nix

```diff
@@ -0,0 +1,55 @@
+{ lib, ... }:
+
+let
+  meilisearchKey = "TESTKEY-naXRkVX7nhvLaGOmGGuicDKxZAj0khEaoOZPeEZafv8w9j8V6aKb0NVdXRChL5kR";
+in
+{
+  name = "sharkey";
+
+  nodes.machine =
+    { pkgs, ... }:
+    {
+      services.sharkey = {
+        enable = true;
+        setupMeilisearch = true;
+        environmentFiles = [ "/run/sharkey/environment" ];
```

Comment:
Minor note, since people might theoretically look at the NixOS test as an example configuration: systemd will automatically clean the created runtime directory, i.e. `/run/sharkey`, after the service is shut down. So in the NixOS test, it appears to work fine since it first manually creates the runtime directory, sets up the environment file and then immediately starts the service. Starting a sharkey a second time doesn't work though.
The config example above uses `/run/secrets` which is fine. (I stumbled upon this during reviewing and first misread /run/secrets` as `/run/sharkey` in the config example, which wouldn't have been ideal as a config example)
Just wanted to drop thi